lib/pbench/server/api/resources/users_api.py

```diff
@@ -68,23 +75,21 @@ def post(self):
         # generate auth token for the valid user
         token = self.auth.encode_auth_token("60", user.id)
 
-        # generate email and send to the user
+        # generates the email
         # Ideally the message should be a template
-        # rendered using render_template and the hostname
-        # needs to be a variable that points opens up
-        # pbench-dashboard
+        # rendered using render_template
         msg = Message(
             subject="Reset your password!",
-            sender=self.config.get("pbench-server", "mailfrom"),
+            sender=self.mailfrom,
             recipients=[email],
-            body=(
-                "Please reset your password at http://localhost:8000/reset_password/"
-                + token
-            ),
+            body=("Please reset your password at " + self.host + "/" + token),
```

Comment:
> `body=("Please reset your password at " + self.host + "/" + token),`

`self.host` isn't ideal here; if the dashboard has been reaching the server through a reverse proxy, we ought to use that same path. This is OK for now, however; I'll convert this comment into an issue, which should be addressed by factoring the reverse proxy header logic out of `/api/v1/endpoints` so that it can be shared here. (Or anywhere else we want to report a URI to a client.)